app/main/views/g_cloud.py

```diff
@@ -340,54 +332,72 @@ def save_search(framework_framework):
         **build_search_query(request.args, filters.values(), content_manifest, lots_by_slug)
     )
 
+    projects = get_direct_award_projects(current_user.id)['open_projects']
+    projects.sort(key=lambda x: x['name'])
+
     return render_template(
         'direct-award/save-search.html',
         framework_framework=framework_framework,
         search_summary_sentence=search_summary.markup(),
         search_api_url=search_api_url,
         form=CreateProjectForm(),
+        projects=projects,
     )
 
 
 @direct_award.route('/<string:framework_framework>/projects/create', methods=['POST'])
 def project_create(framework_framework):
     form = CreateProjectForm()
-    if form.validate_on_submit():
-        name = form.name.data
+    name = form.name.data
+    name_is_invalid = request.form['save_search_selection'] == "new_search" and not name
+
+    if not form.validate_on_submit() or not request.form['save_search_selection'] or name_is_invalid:
```

Comment:
Something about this isn't working for me, in that if I submit the form without doing anything, I'm just getting a 400 but without the form being rendered again.

Reply:
Hey - I added HTML5 validation to stop the frontend from posting without the radio button selected but the issue was that the request did not have the radio button set, therefore, it threw the 400 error.

Reply:
If you pull down - it should work now. @galund

Reply:
Yep, the HTML 5 validation works. Is there some validation server-side for browsers that don't support that? I think the idea of putting all of these fields into one WTForm is probably sound, if that's do-able.